setup.py

```diff
@@ -22,8 +22,11 @@
 from setuptools.command import easy_install  # pylint: disable=unused-import
 from setuptools.command import install_lib  # pylint: disable=unused-import
 
-if sys.version_info.major == 3 and sys.version_info.minor <=5:
-    warnings.warn("You will soon need to upgrade to python 3.6 in order to use the latest version of Astroid.", DeprecationWarning)
+if sys.version_info.major == 3 and sys.version_info.minor <= 5:
```

Comment:
Unrelated to the MR but I think we can now remove that ?

Reply:
Yeah, absolutely. Didn't even notice it while preparing the PR 😅